### ISSUE TITLE
Add a NoopCodec

### ISSUE THREAD

**Issue by Keno** [bicycle1885/TranscodingStreams.jl#2]
For usability, I think it could be useful to have a noop codec, e.g. for use in APIs such as:
```
codec = NoopCodec()
if #= is bzip2 =#
    codec = Bzip2Decompression()
elseif #= is gzip =#
   ...
end
transcode(codec, data)
```

**Comment by bicycle1885** [comment]
What you want may be [`IdentityCodec`](https://github.com/bicycle1885/TranscodingStreams.jl/blob/1844494308554e0188f21a9c0b17d12a37f80d20/src/identity.jl). This is not yet documented and sub-optimal because it does a redundant copy.

**Comment by bicycle1885** [comment]
Closed by #4.
